### DISCUSSION TITLE
[SANDBOX] Your client does not have permission to get URL /auth/authentication/sh_xxx/secret-key from this server.

### DISCUSSION THREAD

**Discussion by Klungstene** [PayGreen/documentation#14] [Q&A] [answered]
Bonjour,

La requête fonctionne super bien dans le pas à pas developers, mais je bloque pour la reproduire depuis mon code...

Déjà, il me demandait un header content-lenght. Ca c'est ok.

Mais maintenant, je ne suis pas autorisé ?? Pourquoi ?

Merci 🙏🏼

**Comment by Klungstene** [comment]
Finalement ca s'est mis à fonctionner... après un café...

**Comment by Klungstene** [comment]
Ah ben non finalement... C'est aléatoire. Fonctionne environ 1 fois sur 3.

:perplexe

**Comment by theotruvelot** [comment, marked as answer]
Bonjour @Klungstene, 

Navré que votre requête ne fonctionne pas et pour le délai de réponse, 

Pouvez-vous me donner votre requête complète, avec par exemple une commande `curl` afin que je puisse vous aider.

Merci d'avance,

Théo.
